source/stylesheets/base/_variables.scss

```diff
@@ -1,18 +1,29 @@
 // Breakpoints
-$breakpoint-medium: 480px;
-$breakpoint-large: 890px;
+$breakpoint-medium: 40em;
+$breakpoint-large: 55em;
 
 // Colors
-$teal: #12939A;
+$teal: #118288;
 $teal-dark: mix($teal, black, 80%);
+$teal-transparent: rgba(17, 130, 136, 0.7);
+$very-light-gray: #F2F2F2;
```

Comment:
We like to keep the case of the hex codes consistent. According to per https://github.com/thoughtbot/guides/tree/master/style/sass, we can go with either case, but I've found that keeping them all lowercase is easier to maintain and read.

Easy way to do this in Sublime:
Highlight text, and then `cmd + k + l`